server/py/framework/service/__init__.py

```diff
@@ -104,6 +123,7 @@ async def _custom_setup_service(self):
 
     async def _teardown_service(self):
         await self._custom_teardown_service()
+        # TODO: Don't cancel periodic of mounted services
```

Comment:
Elaborate?

Reply:
The cancel is called twice:
1. On API teardown
2. On alerts teardown
But it cancels all periodic functions so the 2nd time is void